src/Commands/ConstructCommand.php

```diff
@@ -93,7 +93,8 @@ protected function configure()
         $this->setName('generate');
         $this->setDescription('Generates a basic PHP project');
         $this->addArgument('name', InputArgument::REQUIRED, $nameDescription);
-        $this->addOption('test', 't', InputOption::VALUE_OPTIONAL, $testDescription, 'phpunit');
+        $this->addOption('test', 't', InputOption::VALUE_OPTIONAL, $testFrameworkDescription, 'phpunit');
+        $this->addOption('test-framework', null, InputOption::VALUE_OPTIONAL, $testFrameworkDescription, 'phpunit');
         $this->addOption('license', 'l', InputOption::VALUE_OPTIONAL, $licenseDescription, 'MIT');
```

Comment:
The alias `test-framework` value is not being grabbed. When the command executes the value must be retrieved (if used) ` $input->getOption('test-framework');` I think we should check both `test` and `test-framework` until the `test` option is removed on a major version.